Makefile

```diff
@@ -371,8 +371,10 @@ kind-bootstrap-hosted: kind-install-hosted install-crds install-resources kind-d
 kind-install-hosted: $(ADDON_CONTROLLER)
 	@cd $(ADDON_CONTROLLER) && KIND_VERSION=$(KIND_VERSION) HOSTED_MODE=true ./build/manage-clusters.sh
 	@cp $(ADDON_CONTROLLER)/kubeconfig_cluster1_e2e ./kubeconfig_$(HUB_CLUSTER_NAME)
+	cp ./kubeconfig_$(HUB_CLUSTER_NAME) ./kubeconfig_$(HUB_CLUSTER_NAME)_e2e
```

Comment:
There's an `_e2e` suffix right above this, so I'm not sure this is necessary. Would you point to the error where this is happening?

Reply:
The error is happening at the "Patch Component Image" job of the workflow here: https://github.com/open-cluster-management-io/governance-policy-framework-addon/actions/runs/8255632781/job/22582549257. 

Specifically, at the `kind-deploy-controller-dev` step where it tries to create resources using the hub/managed kubeconfigs. After looking at it again, I could probably set an environment variable in the framework-addon to point to the kubeconfig file that's already created.

Reply:
So the cause is that this PR:
- https://github.com/stolostron/governance-policy-framework-addon/pull/141

Was never submitted to upstream because the framework tests weren't running there, so it wasn't relevant. The `framework-addon` doesn't have this logic, so it's using the wrong code path. Adding the commit to the upstream PR alongside the framework workflow ought to resolve it.

Reply:
Oh perfect, I'll do that. Thanks!